### DISCUSSION TITLE
What's the proper way to use browser specific styles?

### DISCUSSION THREAD

**Discussion by jorisw** [system-ui/theme-ui#1175] [General] [answered]
Hi, 

I'm trying to use `-webkit-text-fill-color` to override disabled input text color on Safari.

Using such a style as part of an `SxStyleProp` however, yields a type error:
```
Type 'string' is not assignable to type 'SystemStyleObject'.
```

How do I use `-webkit-text-fill-color` as part of an `SxStyleProp` ?

My code: 

```ts
export const disabledStyles: SxStyleProp = {
  cursor: 'default',

  '& select': {
    // Reset iOS / Safari disabled text color
    '-webkit-text-fill-color': '#A0A0BC',
  },
}
```

**Comment by jorisw** [comment]
Also asked in #1145

**Comment by jorisw** [comment]
Workaround: 

```ts
const resetDisabledSelectOptionColor: CSSProperties = {
  WebkitTextFillColor: '#A0A0BC',
}


export const disabledStyles: SxStyleProp = {
  cursor: 'default',

  '& select, & option': resetDisabledSelectOptionColor,
}
```

**Comment by hasparus** [comment, marked as answer]
Hey, @jorisw, which version of Theme UI are you using? (This is already fixed in 0.4.0-rc.5)

**Reply by jorisw** [reply]
I am using `0.3.1`, I did try some version of `0.4.0-rc` to no avail, but will be happy to try again.

**Reply by hasparus** [reply]
Yeah, `0.4.0.-rc.3` was broken :face_with_head_bandage:  Tell me how it went if you try `rc.5` :)

This bug is on type-level only, so in case of 0.3.1 the thing we could do would be backporting the change to DefinitelyTyped.

**Reply by jorisw** [reply]
Yep, works with `0.4.0-rc.5` !
